apps/web/src/components/ImageRaw/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { getImageAbsoluteSource } from 'apps/web/src/utils/images';
+import { StaticImageData } from 'next/image';
+import { CSSProperties } from 'react';
+
+type ImageRawProps = {
+  src: string | StaticImageData;
+  alt: string;
+  title?: string;
+  width?: number | `${number}` | undefined;
+  height?: number | `${number}` | undefined;
+  className?: string;
+  onLoad?: React.ReactEventHandler<HTMLImageElement>;
+  style?: CSSProperties;
+};
+
+// This image component is to purposefully avoid loading images via base.org
+export default function ImageRaw({
```

Comment:
Can we just use `img` tags directly instead of adding this component?

Reply:
we could but then we'd have to add 

// eslint-disable-next-line @next/next/no-img-element

everywhere

Reply:
IMO nice to have a component that is responsible / declarative for this usecase

Reply:
Could potentially add logging for non-supported assets so we can see if anybody trying to abuse the CSP.